scripts/store/main/reducer.ts

```diff
@@ -57,8 +57,13 @@ export default function (state = initialState, action: ActionTypes): SessionStat
             break;
         }
         case Constants.ADD_TO_BASKET: {
-            showToastDialog();
-            action.payload.count == 1 ? callVibrate(VIBRATION_ADD_TIME) : callVibrate(VIBRATION_REMOVE_TIME);
+            if (action.payload.count == -1) {
+                removeFromBasketToast();
+                callVibrate(VIBRATION_REMOVE_TIME);
+            } else {
+                addToBasketToast();
+                callVibrate(VIBRATION_ADD_TIME);
+            }
             if (newState.basket.some((pId) => pId._id === action.payload.product._id)) {
                 let updatedData = newState.basket.map((basketItem) =>
```

Comment:
We should use const if the variable will not change in the next lines.

Reply:
DONE.